builds/any/rootfs/buster/common/overlay/etc/systemd/system/switchdev-online@.service

```diff
@@ -0,0 +1,20 @@
+[Unit]
+Description=Online state for switchdev device %i
```

Comment:
Likewise, only one copy should be enough (and effective).

Reply:
What I meant is that you add this (and the service) twice, once at

```
builds/any/rootfs/buster/common/overlay/etc/systemd/system/networking.service.d/switchdev-online.conf
builds/any/rootfs/buster/common/overlay/etc/systemd/system/switchdev-online@.service
```

and a second copy at

```
builds/any/rootfs/buster/common/overlay/etc/systemd/systemd/system/networking.service.d/switchdev-online.conf
builds/any/rootfs/buster/common/overlay/etc/systemd/systemd/system/switchdev-online@.service
```

Looking a tthe paths and strech, the `systemd/systemd` seems to be bogus and can be dropped.

Reply:
oh i see. will fix